lib/framebuffer-internal.h

```diff
@@ -86,6 +86,28 @@ class Framebuffer {
       unsigned int output_enable_rev2 : 1;  // 2      (Pi1.Rev2; masks: I2C SDA)
       unsigned int clock_rev2         : 1;  // 3      (Pi1.Rev2; masks: I2C SCL)
 #endif
+#ifdef ADAFRUIT_RGBMATRIX_HAT // assumes 40 pins (A+/B+/Pi2)
+      unsigned int output_enable      : 1;  // 4
+      unsigned int p0_r1              : 1;  // 5      
+      unsigned int p0_b1              : 1;  // 6      
+      unsigned int unused_7_11        : 5;  // 7-11
+      unsigned int p0_r2              : 1;  // 12     
+      unsigned int p0_g1              : 1;  // 13     
+      unsigned int p1_r1              : 1;  // 14
```

Comment:
Please remove all the p1_xx things as they are not needed. Replace them with `unused_<from-gpio>_<to-gqpio>`

So in this case remove p1_r1, p1_g1 and replace with
```
unused_14_15      : 2
```